### ISSUE TITLE
hide actions of future projects on dashboard

### ISSUE THREAD

**Issue by Kowalevskaja** [roma0104/gsd5#68]
This is a feature request, rather than an issue:
Is it possible to hide actions of future projects from the dashboard? 

I think a project marked "future" should kind of override its actions' "next/waiting/future" field. I like to think of a "future" project also as a "paused" project, that is postponed for a while. Such projects should not mess up the dashboard, which I use for daily business. For daily business only actions of an active project or actions without a corresponding project are relevant.

As an unsatisfactory workaround I could mark all actions of future projects "future", such that they don't occur in "next" and "waiting" sections of the dashboard. However, for future projects it is still useful to already order actions as "next" (first), "waiting" (depending on first steps) and "future" (optional additional steps).

See the screenshot below for a minimal example. In there, according to my proposition, the actions "next-future", "waiting-future" and "future-future" are supposed to not show up on dashboard (as long as the project "future" is marked future).
![screenshot](https://cloud.githubusercontent.com/assets/8849499/9250313/3ba16b22-41cc-11e5-994e-d19f9c48b0ab.png)

I don't care whether this behaviour shall be default or optional via a checkbox setting. 
Thanks for consideration and keep up the great work!

**Comment by roma0104** [comment]
Thanks for the idea.
I will take a look at doing this, and I see the value of hiding such actions.  I definitely see it as an optional feature but it might take some heavy rewriting of the macro that generates these action lists.  (I have been super busy the last few months, and I have slept a few times since I really studied my code...)

**Comment by danielo515** [comment]
I missed this feature myself a couple of times. :+1:

**Comment by Kowalevskaja** [comment]
Many thanks for considering my idea. (And no stress! I am busy myself and
know these situations...)

Im Sträler 40, 8047 Zürich, Schweiz / +41 78 609 37 87 /
rempfler_georg@gmx.ch

2015-08-21 2:56 GMT+02:00 Roma <notifications@github.com>:

**Comment by tobibeer** [comment]
As a comment, I don't think it is useful to think of "waiting" as "waiting for a previous tiddler to complete". That would rather be a dependency. Waiting, to me, means waiting for a certain event to occur, e.g. a response, a shipment, etc...

So, in GTD lingo, the only way you managed to have those tags set is that you were already starting to work on the project. Not? ...which you possibly shouldn't have, seeing as how it ain't active yet.

**Comment by Kowalevskaja** [comment]
Thanks for your advice. You're right. My expression
> "waiting" (depending on first steps)

is unfortunate / bad / wrong. Dependencies are the way to go for such situations. 

However, I still consider my feature request useful. You might be right to say, I already have started "future" projects. I start creating actions as steps to reach a goal, I detect dependencies of steps and I detect events that have to happen first. But this is just a rough pre-structuring. I am not really working on the project on a daily business. So, it is thinkable that all tags next/waiting/future make sense for such actions and I really want the project to be marked "future", also but not exclusively in the sense of "paused and to be worked on further in the future".

**Comment by tobibeer** [comment]
Yes, I failed to voice my agreement, actions of future projects, i.e. those that are not currently active, should not be on the dashboard.
